libs/sdk-ui-kit/src/Header/tests/generateHeaderMenuItemsGroups.test.ts

```diff
@@ -0,0 +1,145 @@
+// (C) 2021 GoodData Corporation
+import { getAccountMenuFeatureFlagsMock, getWorkspacePermissionsMock } from "./mock";
+import { generateHeaderMenuItemsGroups } from "../generateHeaderMenuItemsGroups";
+
+describe("generateHeaderMenuItemsGroups", () => {
+    it("should return dashboards and report item", () => {
```

Comment:
Similar to other tests, the test name is not complete. Looking at the test output woulld be very confusing in this state

```
...
should return dashboards and report item
should not return dashboards and report item
...
```

Please provide more details in the test name.

Reply:
I have added more details for all test case names of this file thanks